content/espanol/csharp-guess-the-word/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "C#: Adivina la palabra "
+description: "Una guía introductoria para aprender C# y para usar tus nuevas habilidades a crear un juego de Adivina la palabra."
+prereq: "Ninguno"
+difficulty: "Intermedio"
+download: ""
+draft: false
+icon: "fas fa-code"
+weight: 3
+translationKey: "chsarp-adivina-la-palabra"
```

Comment:
I wonder if we want the Spanish page URL to point to something more meaningful like:

https://workshops.nuevofoundation.org/csharp-adivina-la-palabra/
 
If so, please rename the folder to csharp-adivina-la-palabra, and rename all the files to use Spanish terms.

Also, the translation key would need to be updated to be the same in the English file:

```suggestion
translationKey: "chsarp-guess-the-word"
```

Reply:
This is a good point if we do that we would also need to change the existing ones in Spanish. When checking, it seems that the Python turtle goes like this: "https://workshops.nuevofoundation.org/**es**/python-turtle/"
How about we keep the /es/ in there so regardless of the language it is consistent? Let me know.

Reply:
Thanks @beagandica for pointing it out. I think we should discuss this as a team before we start having many translations on the website. :)